md_main.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+import logging
+import pandas as pd
+import pathlib
+import shutil
+import sqlite3
+from os import path
+
+from scripts.database_operations import *
+
+# todo: turn all print and error statements into logging
+# test run: python md_main.py metadata -m ./test/data/gordon_db_ready.tsv --db "test.db"
+
+EXPECTED_COLS_MD = [
+    "external_source",
+    "external_accession",
+    "filename",
+    "filepath",
+    "bioproject",
+    "biosample",
+    "wgs_master",
+    "refseq_category",
+    "taxid",
+    "species_taxid",
+    "organism_name",
+    "infraspecific_name",
+    "isolate",
+    "version_status",
+    "assembly_level",
+    "release_type",
+    "genome_rep", 
+    "seq_rel_date",
+    "asm_name",
+    "submitter",
+    "gbrs_paired_asm",
+    "paired_asm_comp",
+    "ftp_path",
+    "excluded_from_refseq",
+    "relation_to_type_material",
+    "asm_not_live_date"
+    ]
+
+INSERT_COLS_IDENTIFIER = [
+    'filename',
+    'filepath',
+    'external_accession',
+    'external_source'
+]
+
+INSERT_COLS_MD = EXPECTED_COLS_MD.copy()
+INSERT_COLS_MD.remove('filename')
+INSERT_COLS_MD.remove('filepath')
```

Comment:
If using the frozenset, this would then be

```suggestion
INSERT_COLS_MD = EXPECTED_COLS_MD - {'filename', 'filepath'}
```